src/js/containers/search/visualizations/rank/SpendingByAwardingAgencyVisualizationContainer.jsx

```diff
@@ -152,7 +152,8 @@ export class SpendingByAwardingAgencyVisualizationContainer extends React.Compon
         // generate the API parameters
         const apiParams = {
             field: 'federal_action_obligation',
-            group: `awarding_agency__${this.state.agencyScope}_agency__name`,
+            group: [`awarding_agency__${this.state.agencyScope}_agency__name`,
+                `awarding_agency__toptier_agency__abbreviation`],
```

Comment:
Why are you always fetching the toptier agency abbreviation? Shouldn't you fetch the subtier agency abbreviation when "Sub-Agencies" is selected?